### DISCUSSION TITLE
How to best integrate Prefect and Sentry

### DISCUSSION THREAD

**Discussion by tonal** [PrefectHQ/prefect#3309] [Q&A]
[Sentry-python](https://github.com/getsentry/sentry-python) has integrate to many frameworks.
For example [airflow](https://docs.sentry.io/platforms/python/guides/airflow/) and [celery](https://docs.sentry.io/platforms/python/guides/celery/).
Is such an integration with the prefect planned?

**Comment by joshmeek** [comment]
Hi @tonal I don't think an official integration has yet been planned but we would definitely be open to community contributions to make this happen!

**Comment by tonal** [comment]
#4672
